src/database/world_graph.lua

```diff
@@ -359,10 +364,11 @@ function plantation:new(worldGraph)
     if not self.world.regions.arthur:canAccess(items) then return false end
     if _has(items, "teleportKey") then return true end
     if self.world.regions.outerWall:canAccess(items) then return true end
+    if _has(items, "bomb") and _has(items, "weaponSN") and self.world.regions.grasstownEast:canAccess(items) then return true end
```

Comment:
I believe this check is already covered by the `outerWall` access check, which in turn checks `eventKazuma`

Reply:
The Outer Wall check requires either `eventKazuma` and `flight` and `eventCore`, or `teleportKey` and Plantation access, but you can get into the Plantation through Kazuma's shelter without either of those requirements.

Reply:
I suppose it might make sense to replace `_has(items, "bomb")` with `_has(items, "eventKazuma")` here, though

Reply:
yeah you can at least replace the bomb and east grasstown check with kazuma check (can't get kazuma without east grasstown access ofc)